_topic_maps/_topic_map.yml

```
@@ -1260,6 +1260,8 @@ Topics:
     File: external-secrets-operator-uninstall
   - Name: External Secrets Operator APIs
     File: external-secrets-operator-api
+  - Name: Migrating from the community External Secret Operator to the Red Hat External Secret Operator
```

I am not sure if this content appears in the doc, but I think we should use legal team approved name.

```suggestion
  - Name: Migrating from the community External Secret Operator to the External Secret Operator for Red Hat OpenShift
```